library/secret.go

```diff
@@ -79,6 +79,8 @@ func (s *Secret) Match(from *pipeline.Container) bool {
 		eACL = checkEvent(events, constants.EventDeploy)
 	case constants.EventComment:
 		eACL = checkEvent(events, constants.EventComment)
+	case constants.EventDelete:
```

Comment:
I think we should implement secrets allowed events the same way we do repo allowed events

Reply:
Can do, let me know if the changes I made are what you imagined